src/app/home-page/widget/organization-box/organization-box.component.html

```diff
@@ -1,80 +1,87 @@
-          <div [ngPlural]="pendingRequests.length">
-            <ng-template ngPluralCase="=1">1 Pending Request</ng-template>
-            <ng-template ngPluralCase="other">{{ pendingRequests.length }} Pending Requests</ng-template>
-          </div>
-        </a>
-      </div>
-      <div id="pending-invites" fxLayout="row" fxLayoutAlign="center center" class="mb-3">
-        <mat-icon class="pending-invites mr-1">error</mat-icon>
-        <a routerLink="/accounts" class="size-small pending-links">
-          <div [ngPlural]="pendingInvites.length">
-            <ng-template ngPluralCase="=1">1 Pending Invitation</ng-template>
-            <ng-template ngPluralCase="other">{{ pendingInvites.length }} Pending Invitations</ng-template>
-          </div>
-        </a>
+  <div fxLayout="column" fxLayoutAlign="start stretch" class="h-100">
+    <div id="header-bar" fxLayout="row wrap" fxLayoutGap="1rem" class="space-between">
+      <div id="organization-number" fxLayout="row">
+        <mat-card-header fxLayout="row">
+          <mat-card-title>Organizations</mat-card-title>
+          <span class="bubble">{{ listOfOrganizations.length }}</span>
+        </mat-card-header>
       </div>
-      <button
-        mat-button
-        class="private-btn small-btn-structure mb-3"
-        id="requestOrganizationButton"
-        matTooltip="Request Organization"
-        matTooltipPosition="after"
-        (click)="requireAccounts()"
-      >
-        <img src="../../../assets/svg/icons-plus.svg" alt="Register an Organization" class="pr-2 pb-1" />
-        Request an Organization
-      </button>
-    </div>
-  </div>
-  <mat-divider></mat-divider>
-  <mat-card-content class="p-3 h-100">
-    <div *ngIf="totalOrgs !== 0 || isLoading; else noOrganizations">
-      <h5 class="weight-bold">Recent Activity</h5>
-      <div *ngFor="let org of listOfOrganizations" fxLayout="row" fxLayoutAlign="space-between center" class="widget-list-items w-100 py-2">
-        <a class="no-wrap" [routerLink]="'/organizations/' + org.name">
-          {{ org.name }}
-        </a>
-        <div class="date-display">{{ org.lastUpdateDate | date: 'MMM d, yyyy' }}</div>
+      <div fxLayout="row wrap" fxLayoutGap="1rem">
+        <div id="pending-requests" fxLayout="row" fxLayoutAlign="center center" class="mb-3">
+          <mat-icon class="pending-requests mr-1">pending_icon</mat-icon>
+          <a routerLink="/accounts" class="size-small pending-links">
+            <div [ngPlural]="pendingRequests.length">
+              <ng-template ngPluralCase="=1">1 Pending Request</ng-template>
+              <ng-template ngPluralCase="other">{{ pendingRequests.length }} Pending Requests</ng-template>
+            </div>
+          </a>
+        </div>
+        <div id="pending-invites" fxLayout="row" fxLayoutAlign="center center" class="mb-3">
+          <mat-icon class="pending-invites mr-1">error</mat-icon>
+          <a routerLink="/accounts" class="size-small pending-links">
```

Comment:
I think this should route to `/requests` instead of `/accounts` so you can view pending invites and requests

Reply:
Ah yes, I have a [ticket](https://ucsc-cgl.atlassian.net/browse/SEAB-5198) which includes this, I think I'll hold off on it just to address all of the small org. card fixes together